compat/mangle.json

```diff
@@ -20,11 +20,12 @@
       "$_root": "__p",
       "$_commit": "__c",
       "$_suspensions": "__u",
+      "$_fallback": "__f",
       "$_diff": "__b",
       "$_render": "__r",
       "$_hook": "__h",
       "$_catchError": "__e",
       "$_unmount": "_e"
     }
   }
-}
\ No newline at end of file
+}
```

Comment:
Nit: This file shouldn't end in a new line. One of the build scripts writes this file after every build and writes it without a final new line. Build all again and you should see a change that removes this final new line (at least it does on my machine 🤷‍♂ ).

Reply:
@andrewiggins
Thank you for review. 
I removed the last empty line.